src/libstd/collections/hash/map.rs

```diff
@@ -34,13 +34,9 @@ use super::table::BucketState::{
     Full,
 };
 
-const INITIAL_LOG2_CAP: usize = 5;
-const INITIAL_CAPACITY: usize = 1 << INITIAL_LOG2_CAP; // 2^5
+const MIN_NONZERO_RAW_CAPACITY: usize = 32;
```

Comment:
Maybe we should leave something to remind that it must be a power of 2? Like the `// 2^5` before